index.html

```diff
@@ -1,20 +1,247 @@
+            alt="hero photo"
+          />
+          <h2 class="d-flex bold-text justify-content-center my-2">
+            Maya Nelson
+          </h2>
+          <p class="d-md-flex d-none justify-content-center text-uppercase">
+            Project Manager
+          </p>
+          <hr class="blue-line" />
+          <!-- hero buttons-->
+          <div class="row text-center d-md-none bold-text mb-4">
+            <div class="col">
+              <a
+                href="resume.html"
+                type="button"
+                class="btn btn-primary resume-btn rounded-pill px-4"
+              >
+                RESUME
+              </a>
+            </div>
+            <div class="col">
+              <a
+                href="projects.html"
+                type="button"
+                class="btn btn-outline-dark projects-btn rounded-pill px-4"
+              >
+                PROJECTS
+              </a>
+            </div>
+          </div>
+          <!-- social media icons row -->
+          <div
+            class="row row-cols-auto bg-white my-3 py-1 justify-content-center social overflow-hidden"
+          >
+            <a href="https://www.facebook.com/" class="col" target="_blank">
+              <img
+                class="social-img"
+                src="https://static.wixstatic.com/media/0fdef751204647a3bbd7eaa2827ed4f9.png/v1/fill/w_40,h_40,al_c,usm_0.66_1.00_0.01/0fdef751204647a3bbd7eaa2827ed4f9.png"
+                alt="facebook icon"
+              />
+            </a>
+            <a
+              href="https://twitter.com/?lang=en-gb"
+              class="col"
+              target="_blank"
+            >
+              <img
+                class="social-img"
+                src="https://static.wixstatic.com/media/c7d035ba85f6486680c2facedecdcf4d.png/v1/fill/w_40,h_40,al_c,usm_0.66_1.00_0.01/c7d035ba85f6486680c2facedecdcf4d.png"
+                alt="twitter icon"
+              />
+            </a>
+            <a href="https://uk.linkedin.com/" class="col" target="_blank">
+              <img
+                class="social-img"
+                src="https://static.wixstatic.com/media/6ea5b4a88f0b4f91945b40499aa0af00.png/v1/fill/w_50,h_50,al_c,usm_0.66_1.00_0.01/6ea5b4a88f0b4f91945b40499aa0af00.png"
+                alt="linkedin icon"
+              />
+            </a>
+            <a href="https://www.instagram.com/" class="col" target="_blank">
+              <img
+                class="social-img"
+                src="https://static.wixstatic.com/media/01c3aff52f2a4dffa526d7a9843d46ea.png/v1/fill/w_40,h_40,al_c,usm_0.66_1.00_0.01/01c3aff52f2a4dffa526d7a9843d46ea.png"
+                alt="instagram icon"
+              />
+            </a>
+          </div>
+        </div>
 
-<body>
+        <div class="d-flex flex-column mx-md-5 py-5 w-100">
```

Comment:
Instead of using `w-100`, use `col-12` here. Make sure this div's parent has the `row` class and apply `col-...` classes to this section and the beige photo block (hero-container) above too.